requirements/requirements_lightning.txt

```diff
@@ -1,4 +1,4 @@
-pytorch-lightning>=1.4.0
+pytorch-lightning>=1.4.8,<1.5
```

Comment:
Let's remove the `<1.5` for main

Reply:
We know that 1.5 has breaking API changes. We should probably relax that after we merge those fixes into main?

Reply:
But how will I know when lightning gets updated to 1.5 if there are no complaints of bugs in our slack channel?